setup.py

```diff
@@ -99,7 +104,55 @@ def gen_packages_items():
     return packages
 
 
+def add_mim_extention():
+    """Add extra files that are required to support MIM into the package.
+
+    These files will be added by creating a symlink to the originals if the
+    package is installed in `editable` mode (e.g. pip install -e .), or by
+    copying from the originals otherwise.
+    """
+
+    # parse installment mode
+    if 'develop' in sys.argv:
+        # installed by `pip install -e .`
```

Comment:
Please check if `pip install -e /path/to/locations/repo`, when cwd is not on the project root, works as intended.

Reply:
I guess L 127 might be affected

Reply:
Good suggestion. Fixed in a new commit.